src/row.ts

```diff
@@ -58,6 +58,7 @@ export interface Family {
 export interface FilterConfigOption {
   method?: string;
   data?: Data;
+  key?: string;
```

Comment:
[this code sample](https://github.com/googleapis/nodejs-bigtable/blob/86c8f4b96668a91f01bda2c75356b1ed12828269/samples/deleteSnippets.js#L88-L103) demonstrates that `key` should be a property of this interface.